### DISCUSSION TITLE
PEP 735 comments about PDM?

### DISCUSSION THREAD

**Discussion by pawamoy** [pdm-project/pdm#2471] [General]
PEP 735 (https://peps.python.org/pep-0735/#projects-are-packages) says things like

> Both PDM and Poetry treat the projects they support as packages. This allows them to use and interact with standard pyproject.toml metadata for some of their needs, and allows them to support installation of the “current project” by doing a build and install using their build backends.
Effectively, this means that neither Poetry nor PDM supports non-package projects.

I wonder if this is actually correct, and if @frostming was in touch with the author :slightly_smiling_face:

**Comment by deronnax** [comment]
It was true until https://github.com/pdm-project/pdm/pull/2394 (undocumented unfortunately)(renamed "distribution" meanwhile). We use it at work and it works very well 👌 .
